File: Java/EstruturaSequencial/Exercicio13.java
// Tendo como dado de entrada a altura (h) de uma pessoa, construa um algoritmo que calcule seu peso ideal, utilizando as seguintes fórmulas:

// a) Para homens: (72.7*h) - 58
// b) Para mulheres: (62.1*h) - 44.7
package EstruturaSequencial;
import java.util.Scanner;

public class Exercicio13 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite a altura da pessoa (em metros): ");
        double altura = scanner.nextDouble();

        double pesoIdealHomens = (72.7 * altura) - 58;
        double pesoIdealMulheres = (62.1 * altura) - 44.7;

        System.out.printf("Peso ideal para homens: %.2f kg%n", pesoIdealHomens);
        System.out.printf("Peso ideal para mulheres: %.2f kg%n", pesoIdealMulheres);

        scanner.close();
    }
}

File: Python/EstruturaSequencial/exercicio_13.py
"""
Tendo como dado de entrada a altura (h) de uma pessoa, construa um algoritmo que calcule seu peso ideal,
utilizando as seguintes fórmulas:

a) Para homens: (72.7*h) - 58
b) Para mulheres: (62.1*h) - 44.7
"""

altura = float(input("Digite a altura da pessoa (em metros): "))
genero = input("Digite o gênero da pessoa (homem/mulher): ")

if genero.lower() == "homem":
    peso_ideal = (72.7 * altura) - 58
elif genero.lower() == "mulher":
    peso_ideal = (62.1 * altura) - 44.7
else:
    print("Gênero inválido. Por favor, insira 'homem' ou 'mulher'.")

print(f"O peso ideal para uma pessoa de {altura:.2f} metros do gênero {genero} é de {peso_ideal:.2f} kg.")
